frontend/server/src/Controllers/Group.php

```diff
@@ -496,6 +512,60 @@ public static function getGroupEditDetailsForTypeScript(
         ];
     }
 
+    /**
+     * @return array{smartyProperties: array{payload: TeamGroupEditPayload, title: \OmegaUp\TranslationString}, entrypoint: string}
+     *
+     * @omegaup-request-param string $team_group_alias
+     */
+    public static function getTeamGroupEditDetailsForTypeScript(
+        \OmegaUp\Request $r
+    ) {
+        // Authenticate user
+        $r->ensureMainUserIdentity();
+
+        $teamGroupAlias = $r->ensureString(
+            'team_group_alias',
+            fn (string $alias) => \OmegaUp\Validators::namespacedAlias($alias)
+        );
+        $teamGroup = self::validateTeamGroupAndOwner(
+            $teamGroupAlias,
+            $r->identity
+        );
+        if (is_null($teamGroup) || is_null($teamGroup->team_group_id)) {
+            throw new \OmegaUp\Exceptions\InvalidParameterException(
+                'parameterNotFound',
+                'team_group_alias'
+            );
+        }
+
+        return [
+            'smartyProperties' => [
+                'payload' => [
+                    'teamGroupAlias' => $teamGroupAlias,
+                    'teamGroupName' => $teamGroup->name,
+                    'teamGroupDescription' => $teamGroup->description,
```

Comment:
```suggestion
                    'teamGroup' => [
                        'alias' => $teamGroupAlias,
                        'name' => $teamGroup->name,
                        'description' => $teamGroup->description,
                    ],
```

se vale tener estructuras anidadas!

Reply:
Se ve mejor así. Gracias por el tip.